service/topology_state_machine.cc

```diff
@@ -203,6 +203,7 @@ static std::unordered_map<global_topology_request, sstring> global_topology_requ
     {global_topology_request::new_cdc_generation, "new_cdc_generation"},
     {global_topology_request::cleanup, "cleanup"},
     {global_topology_request::keyspace_rf_change, "keyspace_rf_change"},
+    {global_topology_request::truncate_table, "truncate_table"},
 };
```

Comment:
docs/dev/topology-over-raft.md

Reply:
I've updated `topology-over-raft.md` with the description of the `truncate_table` in v7. Resolving conversation.